doc/website/getting-started/overview.md

```diff
@@ -40,18 +40,29 @@ To do so, the following lines of code are required
 
 ### Service description
 
-A ``ServiceDescription`` in iceoryx represents the data to be transmitted and is uniquely identified by three string identifiers.
+A ``ServiceDescription`` in iceoryx represents the data to be transmitted and is uniquely identified by three string
+identifiers.
 
 1. ``Group`` name
 2. ``Instance`` name
 3. ``Topic`` name
 
-A triple consisting of such strings is called a ``ServiceDescription``. In Autosar terminology these three
-identifiers are called ``Service``, ``Instance`` and ``Event`` respectively. Note that in the API they are still
-named in this way in the ``capro::ServiceDescription`` class and hence ``Group`` corresponds to and Autosar
-``Service`` and ``Topic`` to an Autosar ``Èvent``.
+A triple consisting of such strings is called a ``ServiceDescription``. The service model of iceoryx is derived
+from AUTOSAR and is still used in the API with these names. The so called canonical protocol is implemented in the
+namespace ``capro``.
 
-Two ``ServiceDescription`` are considered matching if all these three strings are element-wise equal, i.e. group,
+The following tables gives an overview over the different terminologies:
+
+|         | Group   | Instance | Topic |
+|---------|---------|----------|-------|
+| ROS2    | -       | Topic    | Type  |
+| AUTOSAR | Service | Instance | Event |
+| DDS     | -       | Topic    | Type  |
+
```

Comment:
I don't think that ROS2 topic/type and Instance/Event match exactly. Same for DDS. I think we should remove this table.

Reply:
Can you double check @budrus?

Reply:
As discussed we use currentty the group name (aka servive ID) in rmw_iceoryx for storing the topic type and the instance name for storing the topic name with full namespace

Reply:
What is the reason for renaming service -> group ?
This is very confusing to me.
There are three reasons:
* A service is a common concept used in development and quickly and easily understood
* When I think of group, I think of a collection of services I can publish to, not a single one
* The iceoryx group maps to an AUTOSAR service - isn't this confusing?

@MatthiasKillat @budrus

Reply:
Move your remark to #415